doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
I'll leave to your discretion whether it makes sense to support revocation for multiple master public keys.  Trying to put on my customer hat (which I don't wear very well), I kind to think it does.  Once I've added a revoked key to my config, I have very little motivation to remove it.  And if some newbie really botches things and goes through three master keys in as many days we probably ought to have a way to handle that.  But I could be convinced otherwise.